pallets/asset/src/lib.rs

```diff
@@ -426,6 +436,13 @@ decl_storage! {
                 };
                 ClassicTickers::insert(&import.ticker, classic_ticker);
             }
+
+            let sys_did = &config.reserved_country_currency_codes[0].0;
```

Comment:
I thought there may be several systematic identities, in which case we need to handle more than just the first pair.

On the other hand, if the requirements call for just one systematic identity (which I don't think they do) then we don't need the outer vector and we could have 
```
config(reserved_country_currency_codes): (IdentityId, Vec<CountryCurrencyCodes>);
```
or even better, define this systematic identity separately:
```
config(systematic_identity): IdentityId;
config(reserved_country_currency_codes): Vec<CountryCurrencyCodes>;
```
Having said that, I don't personally believe there is a requirement to fix just one identity for ticker symbol reservation. @adamdossa please correct me if I'm mistaken.

Reply:
Good point I would have to see how Adam responds and I would make the change to suit.